### ISSUE TITLE
Error on installation of chatterbot

### ISSUE THREAD

**Issue by froilanimnida** [gunthercox/ChatterBot#2339]
Hello i got this version related error i tried to downgrade and upgrade my wheel version but i got the same error.

      ERROR: Some build dependencies for preshed<2.1.0,>=2.0.1 from https://files.pythonhosted.org/packages/0b/14/c9aa735cb9c131545fc9e23031baccb87041ac9215b3d75f99e3cf18f6a3/preshed-2.0.1.tar.gz conflict with the backend dependencies: wheel==0.42.0 is incompatible with wheel>=0.32.0,<0.33.0.
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× pip subprocess to install build dependencies did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
![image](https://github.com/gunthercox/ChatterBot/assets/61849282/833327e5-00bb-458d-9166-ff53f86394d8)

**Comment by froilaaaaan1** [comment]
> Same issue here in a completely fresh venv with Python 3.9.2. Looks like preshed's wheel is too modern ;)
> 
> ERROR: Some build dependencies for preshed<2.1.0,>=2.0.1 from https://files.pythonhosted.org/packages/0b/14/c9aa735cb9c131545fc9e23031baccb87041ac9215b3d75f99e3cf18f6a3/preshed-2.0.1.tar.gz conflict with the backend dependencies: wheel==0.42.0 is incompatible with wheel>=0.32.0,<0.33.0.

yup i try downgrading the wheel but the issue still persists. hope it'll be fix soon.

**Comment by SuchanaNag002** [comment]
My issue got solved on doing `pip install "chatterbot==1.0.0"]`

**Comment by froilaaaaan1** [comment]
> My issue got solved on doing `pip install "chatterbot==1.0.0"`

alright i will try to install it again very soon, thanks!

**Comment by yanhackcode15** [comment]
> pip install "chatterbot==1.0.0"

that totally worked. I've been trying diff things for hours. this did it. thx. I also use python 3.8 and have also downgraded spacy to 32

**Comment by froilaaaaan1** [comment]
> My issue got solved on doing `pip install "chatterbot==1.0.0"`

For everyone who's facing the same error stated above, kindly run this thing. It worked on my machine.
